store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReader.java

```diff
@@ -88,6 +89,14 @@ public static CarbonReaderBuilder builder(String tablePath, String tableName) {
     return new CarbonReaderBuilder(tablePath, tableName);
   }
 
+  /**
+   * Return a new {@link CarbonReaderBuilder} instance
+   */
+  public static CarbonReaderBuilder builder(String tablePath) {
+    String uniqueName = "_temp" + UUID.randomUUID().toString();
```

Comment:
Suggest to use a more meaningful name, like `s"table@$tablePath"`

Reply:
I think it maybe has many problems if using s"table@$tablePath", because there are different table path, like local,HDFS and OBS and so on.
I think we can add time to  table name for recording the create time of table.